index.rst

```diff
@@ -1,3 +1,7 @@
+.. meta::
+  :description: Gauge is a light-weight cross-platform test automation tool with the ability to author test cases in the business language.
```

Comment:
Gauge is a light-weight cross-platform test automation framework for writing user acceptance tests in Markdown format.